src/content/docs/Company/security.md

```diff
@@ -0,0 +1,186 @@
+
+## Contractor Security & Confidentiality
+
+Shorebird does not use contractors at this time. If we do in the future, they
+will be covered by the same security policies and processes as employees and
+sign the same confidentiality agreement.
+
+## Internal Access Control
+
+### Network
+
+Shorebird does not run it's own internal network. Given that we are an all
+[remote company](/company/company_details/#a-note-on-remote-only) this is of no
+benefit to us and instead rely on Cloud Services.
+
+We use HTTPS for all communication between our employees, our products, and our
+customers. We use Google Cloud's managed SSL certificates for this.
+
+### Devices
+
+All employees are provided with company devices for work. If you believe you
+need to use an external device for job related functions please work with the
+management team to understand the need and purchase the needed equipment via the
+company.
+
+Employees are obligated to maintain physical security of their devices and the
+confidentiality of any customer data they may access as part of supporting a
+customer.
+
+Devices owned and issued by the company and are required to use full disk
+encryption.
+
+We do not permit access to Shorebird production systems or customer data from
+personal devices.
+
+The vast majority of Shorebird's code and operations are handled in the open,
+either via public code on GitHub, or public discussions on Discord, so while we
+do not permit access to internal systems from personal devices, employees
+generally do not need access to internal systems to do their work.
+
+### Passwords
+
+All passwords are stored in a password manager. We use
+[1Password](https://1password.com/). We do not store passwords in code, in
+emails, or in any other insecure location. Passwords should never be passed to a
+command line program as an argument, as they will then be stored in the shell
+history file.
+
+We do not recommend ever typing a password. Instead, we recommend using a
+password manager to generate and store passwords. This ensures that passwords
+will never be input into a place they were not intended to be.
+
+Access to our SSO provider (Google) has required strong passwords (Google's
+guidelines) and two factor authentication (2FA).
+
+### Physical Access
+
+Shorebird is an
+[all-remote company](/company/company_details/#a-note-on-remote-only). We have
+no physical office or visitor policies. We do not own or operate physical
+servers.
+
+### User Access Review
+
+We review all user access to our systems periodically, as well as as part of an
+employee joining or leaving the company. All access to Shorebird systems is
+gated through Google SSO including required two factor authentication.
+
+## Third Party Suppliers
+
+We use a number of third party services to run our business. We list those which
+may come in contact with customer data as part of our privacy policy:
+https://shorebird.dev/privacy
+
+### Third Party Service Accounts
```

Comment:
Confirming this is not duplicated in the other doc?

Reply:
Good catch. I think having it in the handbook makes more sense as it's an overarching statement about services we use.

Reply:
Removed from docs via https://github.com/shorebirdtech/docs/pull/357